src/Native/Bootstrap/CMakeLists.txt

```diff
@@ -9,5 +9,9 @@ add_compile_options(-Wno-unused-private-field)
 add_compile_options(-Wno-tautological-undefined-compare)
 endif()
 
-add_subdirectory(base)
+# We don't need base for WASM.
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Emscripten)
```

Comment:
Do we have the WASM build variable available here? Based on some of the other comments, it's not clear if emcmake is a long term plan, so it might be better to switch based on the target rather than the cmake name